jenkins_job_wrecker/modules/publishers.py

```diff
@@ -148,10 +155,53 @@ def buildtrigger(top, parent):
+                            elif subconf.tag == 'hudson.plugins.parameterizedtrigger.FileBuildParameters':
+                                for parameter in subconf:
+                                    if parameter.tag == "propertiesFile":
+                                        project["property-file"] = parameter.text
+                                    elif parameter.tag == "failTriggerOnMissing":
+                                        project["fail-on-missing"] = get_bool(parameter.text)
+                                    else:
+                                        raise NotImplementedError("cannot handle FileBuildParameters sub-element %s"
+                                                                  % parameter.tag)
+                            elif subconf.tag == 'hudson.plugins.git.GitRevisionBuildParameters':
+                                git_revision = {}
+                                for parameter in subconf:
+                                    if parameter.tag == "combineQueuedCommits":
+                                        git_revision["combine-queued-commits"] = get_bool(parameter.text)
+                                    else:
+                                        raise NotImplementedError("cannot handle GitRevisionBuildParameters "
+                                                                  "sub-element %s " % parameter.tag)
+                                    project["git-revision"] = git_revision
+                            elif subconf.tag == 'hudson.plugins.parameterizedtrigger.matrix.MatrixSubsetBuildParameters':
+                                for parameter in subconf:
+                                    if parameter.tag == "filter":
+                                        project["restrict-matrix-project"] = parameter.text
+                                    else:
+                                        raise NotImplementedError("cannot handle MatrixSubsetBuildParameters "
+                                                                  "sub-element %s " % parameter.tag)
+                            elif subconf.tag == 'org.jvnet.jenkins.plugins.nodelabelparameter.parameterizedtrigger.NodeLabelBuildParameter':
+                                for element in subconf:
+                                    if element.tag == "name":
+                                        project["node-label-name"] = element.text
+                                    elif element.tag == "nodeLabel":
+                                        project["node-label"] = element.text
+                                    else:
+                                        raise NotImplementedError("cannot handle NodeLabelBuildParameter "
+                                                                  "sub-element %s " % parameter.tag)
+                            elif subconf.tag == 'hudson.plugins.parameterizedtrigger.CurrentBuildParameters':
+                                current_parameters_exist = True
```

Comment:
Or maybe you should set `project["current-parameters"] = True` directly to avoid a temporary variable?

Reply:
But then I wouldn't be able to put
```
current-parameters: false
```

would I?

Reply:
No I cannot because it's for each subproject.

Reply:
I was actually thinking about:

    project = {
        "current-parameters": False,
        "node-parameters": False
    }

then here when encountering `CurrentBuildParameters` just do `project["current-parameters"] = True`. Same for `node-parameters`...

Wouldn't this work? If not let's have a call on this.